streamparse/cli/submit.py

```diff
@@ -4,25 +4,33 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
+import importlib
 import os
-import re
 import sys
 import time
 
-from invoke import run
-from six import string_types
+import simplejson as json
```

Comment:
`simplejson` is much faster on Python 2, and slightly faster on Python 3.  We need to use it for message deserialization, so we want it to be fast, and since we can assume people have it installed, might as well use it here too.